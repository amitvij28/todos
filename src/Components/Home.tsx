import React from "react";
import { Grid, Divider } from "@material-ui/core";
import OptionPanel from "./OptionPanel";
import DisplayPanel from "./DisplayPanel";

const Home = () => {
    return (
        <>
            <div>
                {" "}
                <Grid container>
                    <Grid item xs={12}>
                        <OptionPanel />
                    </Grid>
                    <Grid item xs={12}>
                        <br />
                        <Divider />
                        <br />
                    </Grid>
                </Grid>
                <DisplayPanel />
            </div>
        </>
    );
};

export default Home;
