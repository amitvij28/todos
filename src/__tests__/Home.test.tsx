import React from "react";
import { shallow } from "enzyme";
import Home from "../Components/Home";

describe("Home Component", () => {
    const component = shallow(<Home />);
    it("should render", () => {
        expect(component).toMatchSnapshot();
    });
});
