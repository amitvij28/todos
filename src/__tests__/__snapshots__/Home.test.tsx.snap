// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render 1`] = `
<Fragment>
  <div
    className="makeStyles-root-1"
  >
    <OptionPanel />
    <br />
    <div>
      <DisplayPanel />
    </div>
  </div>
</Fragment>
`;
